core/src/main/resources/error/error-classes.json

```
@@ -22,8 +22,8 @@
   "CANNOT_UP_CAST_DATATYPE" : {
     "message" : [ "Cannot up cast %s from %s to %s.\n%s" ]
   },
-  "CANNOT_USE_MIXTURE" : {
-    "message" : [ "Cannot use a mixture of aggregate function and group aggregate pandas UDF" ]
+  "INVALID_PANDAS_UDF_PLACEMENT" : {
+    "message" : [ "The group aggregate pandas UDF %s cannot be invoked in the same SELECT list as other, non-pandas aggregate functions." ]
```

```suggestion
    "message" : [ "The group aggregate pandas UDF %s cannot be invoked together with other, non-pandas aggregate functions." ]
```

Since we have DataFrame API too.